src/main/resources/templates/mopslayout.html

```diff
@@ -11,10 +11,19 @@
 </head>
 <body>
 <header class="header header-main">
-  <input checked class="button button-sidebar button-sidebar-hide" name="button-sidebar" type="radio">
-  <input class="button button-sidebar button-navigation-primary-show" name="button-sidebar" th:if="${navigation}"
-         type="radio">
-  <input class="button button-sidebar button-sidebar-show" name="button-sidebar" type="radio">
+  <label>
```

Comment:
Sämtliche Labels sind falsch eingebettet, nicht nötig und machen zudem die CSS und damit die Animationen kaputt.

Reply:
Gibt es eine Möglichkeit die labels trotzdem einzubinden?

Reply:
Ja schon, aber sie sind nutzlos, da sie ja nicht angezeigt werden. Die Radio-Buttons sind nur dazu da, um JavaScript per CSS zu realisieren.

Reply:
Sobald wir WebComponents drin haben, wird das alles dann in JavaScript ausgelagert... auch, wenn das dann heißt, dass es nicht mehr ohne JS funktioniert.

Reply:
Alles klar, dann nehme ich die wieder raus. :+1:

Reply:
Versteh mich nicht falsch. An sich sollte ein Radio-Button sowas immer haben. Dann aber lieber `label` und `input` nebeneinander (und mit Attribut `for` arbeiten) als ineinander verschachtelt. Ineinander wäre semantisch falsch, da du ja dann sagst, dass der `input` zur Beschriftung gehört.

Reply:
Ah, alles klar. Dann wäre `label` mit `for` erstmal eine valide Lösung?

Reply:
Nein, in diesem Fall nicht, da, wie gesagt, die Radio-Buttons in diesem Fall nur ein "Hack" sind. Außerdem werden die Beschriftungen ja nie angezeigt.

Generell sollte aber immer mit `label` und `for` gearbeitet werden... vor allem in Formularen.

Reply:
Guck dir mal styles.scss an und such nach `~` oder `:checked`. Dann weißt du, was ich meine.

Reply:
Alles klar. Jetzt hab ich das auch gecheckt :D